src/tests/resistance/fuzzing.rs

```diff
@@ -398,6 +539,50 @@ async fn fuzzing_slightly_corrupted_messages_during_handshake_responder_side() {
     node.stop().await;
 }
 
+#[tokio::test]
+async fn fuzzing_slightly_corrupted_messages_post_handshake() {
+    // ZG-RESISTANCE-005 (part 4)
+    //
+    // zebra: sends getdata and ignores message.
+    // zcashd: tbd.
+
+    let test_messages = vec![
+        Message::MemPool,
+        Message::Verack,
+        Message::Ping(Nonce::default()),
+        Message::Pong(Nonce::default()),
+        Message::GetAddr,
+        Message::Addr(Addr::empty()),
```

Comment:
I wonder if we shouldn't add variants with non-empty contents as well (where an element is corrupted).

Reply:
Yes, good idea, will add as a follow up 👍